### PR TITLE
Add is.lowerCase & is.upperCase

### LINKED ISSUE #24: Add support for 'is.upperCase'
Checks if the given string is uppercase.

```js
is.upperCase('UPPERCASE'); // true
is.upperCase('camelCase'); // false
is.upperCase('lowercase'); // false
```

**Update README.md**: under `is > String`

### LINKED ISSUE #25: Add support for 'is.lowerCase'
Checks if the given string is lowercase.

```js
is.lowerCase('lowercase'); // true
is.lowerCase('camelCase'); // false
is.lowerCase('UPPERCASE'); // false
```

**Update README.md**: under `is > String`

### PR DESCRIPTION
**Pull request by lucas-carl**
Fixes #24 
Fixes #25

### CODE REVIEW

File: src/is/types.js [hunk 1]
```diff
@@ -125,6 +125,22 @@ export function isWindowObject(value) {
     return value != null && typeof value === 'object' && 'setInterval' in value;
 }
 
+/**
```
Comment (joeltankam) [inline src/is/types.js]: This should not be done here, but in `regexp.js` as mentioned in the issue. 
Checkout @boristane pull request #40

File: src/is/types.js [hunk 2]
```diff
@@ -125,6 +125,22 @@ export function isWindowObject(value) {
...
+ * Returns whether a given value is lower case
+ * @param {*} value the value to check
+ */
+export function isLowerCase(value) {
+    return value != null && typeof value === 'string' && value === value.toLowerCase();
```
Comment (joeltankam) [inline src/is/types.js]: Use methods from `types.js`

File: test/is/types.test.js [hunk 3]
```diff
@@ -197,3 +197,27 @@ describe('isWindowObject', () => {
     });
     testFalsyWithNullable(is.windowObject);
 });
+
```
Comment (joeltankam) [inline test/is/types.test.js]: Same as implementation, should be moved.

File: src/is/types.js [hunk 4]
```diff
@@ -125,6 +125,22 @@ export function isWindowObject(value) {
...
+}
+
+/**
+ * Returns whether a given value is upper case
+ * @param {*} value the value to check
+ */
+export function isUpperCase(value) {
+    return value != null && typeof value === 'string' && value === value.toUpperCase();
```
Comment (joeltankam) [inline src/is/types.js]: Use methods from `types.js`

**Comment by lucas-carl** [comment]
I have updated the file structure. The new methods are now in `regexp.js`

File: src/is/regexp.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import { isString } from './types';
+
+/**
+ * Returns whether a given value is lower case
+ * @param {*} value the value to check
+ */
```
Comment (joeltankam) [inline src/is/regexp.js]: Please add `@returns`

File: src/is/regexp.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+export function isLowerCase(value) {
+    if (!isString(value)) return false;
+    return value === value.toLowerCase();
+}
+
+/**
+ * Returns whether a given value is upper case
+ * @param {*} value the value to check
+ */
```
Comment (joeltankam) [inline src/is/regexp.js]: Please add `@returns`

**Comment by joeltankam** [comment]
Thanks for your contribution :tada:

**Pull request merged**